src/session/sessionReducer.js

```diff
@@ -37,6 +37,11 @@ export type SessionState = {|
   isActive: boolean,
   isHydrated: boolean,
   lastNarrow: ?Narrow,
+
+  /**
+   * Whether the /register request is in progress; i.e., on startup, or on
+   * re-init following a dead event queue after 10 minutes of inactivity.
+   */
   loading: boolean,
```

Comment:
A good example for putting into the "summary line, then prose" style we discussed yesterday in the context of `ZulipVersion`. :slightly_smiling_face: